src/PackageManagement.UI/PackageLoader.cs

```diff
@@ -226,10 +226,22 @@ private IEnumerable<string> GetSupportedFrameworks()
             var localResource = await _packageManager.PackagesFolderSourceRepository
                 .GetResourceAsync<UIMetadataResource>();
 
-            var metadataResource =
+            // UIMetadataResource may not be available
+            // Given that this is the 'Installed' filter, we ignore failures in reaching the remote server
+            // Instead, we will use the local UIMetadataResource
+            UIMetadataResource metadataResource;
+            try
+            {
+                metadataResource =
                 _sourceRepository == null ?
                 null :
                 await _sourceRepository.GetResourceAsync<UIMetadataResource>();
+            }
+            catch (Exception)
```

Comment:
Don't catch Exception. Either do an empty catch, or catch the specific exception we want to get.

Also please open up a follow-up issue so we can clean this up nicely and do the right thing for the description and the icons on the package (because it shouldn't show check if the package *might have an update, instead it should show no icons).